manifests/certonly.pp

```diff
@@ -32,6 +34,7 @@
 #
 define letsencrypt::certonly (
   $domains              = [$title],
+  $custom_plugin        = false,
```

Comment:
can you use the new Pupept 4 Datatype Boolean instead of the validate_re down below?